### DISCUSSION TITLE
How to configure docusaurus typedoc plugin to automatically remove old generated files?

### DISCUSSION THREAD

**Discussion by krutoo** [tgreyuk/typedoc-plugin-markdown#329] [Help] [answered]
Hi, first of all thanks for great plugin.

in my project i want to automatically remove old generated files, for example when i change exclude option.

It is possible to do this by configure docusaurus or `docusaurus-plugin-typedoc`?

**Comment by tgreyuk** [comment, marked as answer]
you should use `cleanOutputDir` - actually this should probably be `true` by default.
